internal/types/action/action.go

```diff
@@ -107,3 +107,16 @@ func (a Type) String() string {
 		"remove-accounts",
 	}[a]
 }
+
+type Actions []Type
```

Comment:
mind adding godocs for the new exported type and its Strings() receiver function?

Reply:
Done.